workers/data_refinery_workers/processors/create_quantpendia.py

```diff
@@ -38,13 +46,24 @@ def download_files(job_context: Dict) -> Dict:
     for key, samples in job_context['samples'].items():
         outfile_dir = job_context['output_dir'] + key + '/'
         os.makedirs(outfile_dir, exist_ok=True)
+
+        logger.debug("Downloading quant.sf files for quantpendia.",
+                    accession_code=key,
```

Comment:
Won't `key` be a species?

Reply:
no, in the case of the quantpendia we are aggregating by experiment